package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A declarative Svelte routing library with SSR support",
   "main": "src/index.js",
   "svelte": "src/index.js",
+  "types": "index.d.ts",
```

Comment:
Maybe the types should be moved to a sub directory, so that when the need arises to add more `.d.ts` files, they are not scattered around the project root... What do you think?

Reply:
Great idea! this sounds very good in terms of scalability

Reply:
Done on https://github.com/EmilTholin/svelte-routing/pull/117/commits/9cba58836e68cd6967bbbefa18405585f6b5ef89, just a quick notification, i also had to install svelte as a `devDependency`, due to some weird bug preventing the `svelte` package from being found in `node_modules`, this happened right after i installed `svelte2tsx` as a `devDependency`, but it's already solved on https://github.com/EmilTholin/svelte-routing/pull/117/commits/bfd9dd4fb1b4884b5761210e1410962cd45d8a5d